examples/ccvm_boxqp_dl.py

```diff
@@ -30,13 +30,13 @@
 
         # Solve the problem using the Adam algorithm
         adam_parameters = AdamParameters(
-            alpha=0.001, beta1=0.9, beta2=0.999, add_assign=False
+            alpha=0.001, beta1=0.9, beta2=0.999, # 1.0, # 
+            add_assign=False
         )
 
         solution = solver(
             instance=boxqp_instance,
-            post_processor=None,
-            algorithm_parameters=adam_parameters,  # Set to None to use the original DL algorithm
+            post_processor=None, # algorithm_parameters=adam_parameters,  # Set to None to use the original DL algorithm
```

Comment:
Let's split the commented code on the next line.

Reply:
Here is the screenshot of what I have done. Hopefully, the change will address your concern. If not let me know.
![Screenshot 2023-11-20 at 5 26 05 PM](https://github.com/1QB-Information-Technologies/ccvm/assets/82167289/17637ca5-3e4d-42b3-a9ed-1a3825209021)